### ISSUE TITLE
Error on Meteor 1.4.2

### ISSUE THREAD

**Issue by tzapu** [studiointeract/accounts-ui#74]
After updating to Meteor 1.4.2 i get the following error:

```
TypeError: null is not an object (evaluating '_state3.formState')
```

running `meteor --release=1.4.1.3` works fine

![screenshot 2016-10-26 12 24 14](https://cloud.githubusercontent.com/assets/2983312/19720540/3d8f4e3a-9b77-11e6-8d75-a9ab45e1fc66.png)

Thank you for a wonderful package

**Comment by SachaG** [comment]
If this is the same issue, I already fixed it with my last commit. We just need @timbrandin to publish the new version.

**Comment by timbrandin** [comment]
This should be solved, for me it's working in 1.2.9.
